OpenShiftTemplate.yml

```diff
@@ -184,3 +181,5 @@ parameters:
   value: quay.io/openshiftio/rhel-fabric8-services-fabric8-tenant
 - name: IMAGE_TAG
   value: latest
+- name: f8_automated_update_enabled
```

Comment:
don't you want to have upper case `F8_AUTOMATED_UPDATE_ENABLED` as in case of other parameters.

Reply:
I followed the same approach as is already used in UI repo: https://github.com/fabric8-ui/fabric8-ui/blob/master/openshift/fabric8-ui.app.yaml#L171 https://github.com/openshiftio/saas-openshiftio/blob/master/dsaas-services/fabric8-ui.yaml#L11

Reply:
well, I would have preferred that we remain consistent within this template, hence using `AUTOMATED_UPDATE_ENABLED` (no `f8_` prefix and uppercased) would have been better, IMO.